tests/src/Kernel/Access/OgGroupContentOperationAccessTest.php

```diff
@@ -0,0 +1,357 @@
+<?php
+
+namespace Drupal\Tests\og\Kernel\Access;
+
+use Drupal\comment\Entity\CommentType;
+use Drupal\Component\Utility\Unicode;
+use Drupal\entity_test\Entity\EntityTest;
+use Drupal\KernelTests\KernelTestBase;
+use Drupal\node\Entity\NodeType;
+use Drupal\og\Entity\OgMembership;
+use Drupal\og\Entity\OgRole;
+use Drupal\og\Og;
+use Drupal\og\OgGroupAudienceHelper;
+use Drupal\og\OgMembershipInterface;
+use Drupal\og\OgRoleInterface;
+use Drupal\user\Entity\User;
+
+/**
+ * Test access to group content operations for group members.
+ *
+ * @group og
+ */
+class OgGroupContentOperationAccessTest extends KernelTestBase {
+
+  /**
+   * {@inheritdoc}
+   */
+  public static $modules = [
+    'comment',
+    'entity_test',
+    'field',
+    'node',
+    'og',
+    'system',
+    'user',
+  ];
+
+  /**
+   * An array of test users.
+   *
+   * @var \Drupal\user\Entity\User[]
+   */
+  protected $users;
+
+  /**
+   * A test group.
+   *
+   * @var \Drupal\entity_test\Entity\EntityTest
+   */
+  protected $group;
+
+  /**
+   * The bundle ID of the test group.
+   *
+   * @var string
+   */
+  protected $groupBundle;
+
+  /**
+   * An array of test roles.
+   *
+   * @var \Drupal\og\Entity\OgRole[]
+   *   Note that we're not using OgRoleInterface because of a class inheritance
+   *   limitation in PHP 5.
+   */
+  protected $roles;
+
+  /**
+   * An array of test group content, keyed by bundle ID and user ID.
+   *
+   * @var \Drupal\Core\Entity\ContentEntityInterface[][]
+   */
+  protected $groupContent;
+
+  /**
+   * The entity type manager.
+   *
+   * @var \Drupal\Core\Entity\EntityTypeManagerInterface
+   */
+  protected $entityTypeManager;
+
+  /**
+   * {@inheritdoc}
+   */
+  protected function setUp() {
+    parent::setUp();
+
+    $this->installConfig(['og']);
+    $this->installEntitySchema('entity_test');
+    $this->installEntitySchema('comment');
+    $this->installEntitySchema('node');
+    $this->installEntitySchema('og_membership');
+    $this->installEntitySchema('user');
+    $this->installSchema('system', 'sequences');
+
+    $this->entityTypeManager = $this->container->get('entity_type.manager');
+
+    $this->groupBundle = Unicode::strtolower($this->randomMachineName());
+
+    // Create a test user with UID 1. This user has universal access.
+    $this->users['uid1'] = User::create(['name' => $this->randomString()]);
+    $this->users['uid1']->save();
+
+    // Create a user that will serve as the group owner. There are no special
+    // permissions granted to the group owner, so this user can be used for
+    // creating entities that are not owned by the user under test.
```

Comment:
> There are no special permissions granted to the group owner

just as a note, this will be eventually like in D7, controlled by a variable (e.g. `Group owner has full access`)